src/canvas-extractor/edfi_canvas_extractor/client_graphql.py

```diff
@@ -193,3 +190,96 @@ def extract_assignments(
     )
 
     return (assignments, udm_assignments_dfs)
+
+
+def extract_submissions(
+    gql: GraphQLExtractor,
+    sync_db: sqlalchemy.engine.base.Engine,
+) -> None:
+    # ) -> Dict[Tuple[str, str], DataFrame]:
+    """
+    Gets all Canvas submissions for sections, in the Ed-Fi UDM format.
+    Parameters
+    ----------
+    sections: List[Dict[str, str]]
+        A List of Section dictionaries.
+    sync_db: sqlalchemy.engine.base.Engine
+        Sync database connection.
+    Returns
+    -------
+    Dict[Tuple[str, str], DataFrame]
+        A dict with (section_id, assignment_id) as key and udm_submissions
+        as value.
+    """
+    export: Dict[Tuple[str, str], DataFrame] = {}
+    submissions: List[Dict[str, str]] = gql.get_submissions()
+    if len(list(submissions)) < 1:
+        logger.info(
+            "Skipping submissions for section id %s - No data returned by API",
```

Comment:
SHOULD: We have to either provide the section id here or reformat this message in this log message to not use `%s`.

Reply:
SUGGESTION:  "Skipping submissions for section - No data returned by API" ?